directory/assets/configuration/spec.yaml

```
@@ -0,0 +1,101 @@
+name: Directory
+files:
+- name: directory.yaml
+  options:
+  - template: init_config
+    options:
+    - name: directory
+      description: |
+        The directory to monitor.
+
+        On Windows, please make sure you escape back-slashes.
+        Otherwise, the YAML parser will fail (e.g "C:\\Users\\foo\\Downloads").
+      value:
+        type: string
+      required: true
+    - name: name
+      description: Tag metrics with a specified name.
+      value:
+        type: string
+        example: <DIRECTORY>
+    - name: dirtagname
+      description: |
+        The name of the key for the tag used for the directory, the value is the value of the `name` option.
+        The resulting tag attached is "<dirtagname>:<name>".
```

```suggestion
        The resulting tag attached is "<DIRECTORY_TAG_KEY>:<DIRECTORY>".
```
Could be helpful to have the example values as the resulting tag in the description?